build-deb

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+#
+# Debian/Ubuntu Packaging Scripts
+# Copyright (C) 2002-2021 by Thomas Dreibholz
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Contact: dreibh@iem.uni-due.de
+
+# Bash options:
+set -e
```

Comment:
I do not know if there is an overall policy here for this repository, but most p4.org code in the p4lang organization is released under the Apache 2.0 license.  I doubt having a single build file like this under GPL would necessarily cause an issue, but wanted to bring it to the attention of @antoninbas in case there is a policy.

Reply:
And same comment for other files proposed to be added in this PR with GPL license.

Reply:
Apache 2.0 is fine as well. I just updated the scripts' headers.